index.html

```diff
@@ -2342,6 +2333,50 @@
           </li>
         </ol>
       </section>
+      <section>
+        <h2>
+          Filter payment options algorithm
+        </h2>
+        <p>
+          The <dfn data-lt="filter payment options">filter payment options
+          algorithm</dfn> runs when user agent presents the user with the
+          collection of payment options available for selection. It MUST run the
+          following steps:
+        </p>
+        <ol>
+          <li>Let <var>matchingOptions</var> be an empty list.</li>
+          <li>Let <var>request</var> be the <a>PaymentRequest</a> object that
+          the user is interacting with.
+          <li>For each <var>requestMethod</var> in
+            <var>request</var>.<a>[[\serializedMethodData]]</a>:
+            <ol type="I">
+              <li>Let <var>filters</var> be <var>requestMethod.data</var>
+                members whose values are lists of strings.</li>
```

Comment:
Hrm. I thought, after our last discussion on this point, we had reached agreement that this was problematic, in that it precluded payment methods from defining `data` parameters whose values were a list of strings, but which were *not* filters.